www/src/pages/docs/doc-links.yaml

```diff
@@ -6,6 +6,7 @@
     Querying with GraphQL*: /docs/querying-with-graphql/
     Gatsby Lifecycle APIs: /docs/gatsby-lifecycle-apis/
     Deploying: /docs/deploy-gatsby/
+    Gatsby Starters: /docs/gatsby-starters
```

Comment:
Just make this "Starters". Also remove "Gatsby" from "Gatsby Lifecycle APIs". The Gatsby part is obvious in these.